### DISCUSSION TITLE
Turn off the X-Amz-Security-Token

### DISCUSSION THREAD

**Discussion by nocjestpiekna** [minio/minio#14435] [Q&A] [answered]
When presigning s3 object from a bucket Minio adds the X-Amz-Security-Token JWT token. How can I turn it off?

**Comment by vadmeste** [comment, marked as answer]
https://docs.aws.amazon.com/general/latest/gr/sigv4-add-signature-to-request.html

```
You can use temporary security credentials provided by the AWS Security Token Service (AWS STS) to sign a request.
The process is the same as using long-term credentials, but requires an additional HTTP header or query string
parameter for the security token.
```

@nocjestpiekna you are signing the request using an STS account. This cannot be turned off as per S3 specification.

**Reply by nocjestpiekna** [reply]
@vadmeste My acquaintance has a Minio instance without a token and he also set up his own accessKey and secretKey to presign requests. Also, I dont want to use sts, but my own credentials.  How this is done?

**Comment by nocjestpiekna** [comment]
Is this possible?

**Reply by harshavardhana** [reply]
> Is this possible?

no
